geotrek/common/tests/test_forms.py

```diff
@@ -0,0 +1,19 @@
+import json
+from django.test import TestCase
+from geotrek.common.forms import HDViewPointAnnotationForm
+from geotrek.common.tests.factories import HDViewPointFactory, LicenseFactory
+
+
+class HDViewPointAnnotateFormTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.vp = HDViewPointFactory(content_object=LicenseFactory())
```

Comment:
you create an HDViewpint whit License instance as related object ? it's not trek / site / poi